components/match2/commons/starcraft_starcraft2/match_summary_starcraft.lua

```diff
@@ -24,7 +25,7 @@ local Opponent = OpponentLibraries.Opponent
 local OpponentDisplay = OpponentLibraries.OpponentDisplay
 
 local ICONS = {
-	greenCheck = '<i class="fa fa-check forest-green-text" style="width: 14px; text-align: center" ></i>',
+	greenCheck = Icon.makeIcon{iconName = 'check', color = 'forest-green-text', size = 'initial'},
 	yellowLine = '<i class="fas fa-minus bright-sun-text" style="width: 14px; text-align: center" ></i>',
 	yellowQuestionMark = '[[File:YellowQuestionMark.png|14x14px|link=]]',
 	redCross = '<i class="fas fa-times cinnabar-text" style="width: 14px; text-align: center" ></i>',
```

Comment:
we should imo replace the cross and line ones too

Reply:
Yeah, but leaving it OOS for this one